indico/modules/events/registration/__init__.py

```diff
@@ -77,9 +78,22 @@ def _inject_event_header(event, **kwargs):
                 .order_by(db.func.lower(RegistrationForm.title))
                 .all())
 
-    participants = get_unique_published_registrations(event)
+    # A participant could appear more than once in the list in case he register to multiple registration form.
+    # This is deemed very unlikely in the case of meetings and lectures and thus not worth the extra complexity.
+    registrations = (Registration
+                     .find(Registration.is_active,
+                           ~RegistrationForm.is_deleted,
+                           RegistrationForm.event_id == event.id,
+                           RegistrationForm.publish_registrations_enabled,
+                           _join=Registration.registration_form)
+                     .order_by(db.func.lower(Registration.first_name),
```

Comment:
We have a classproperty for this so you can do `.order_by(*Registration.order_by_name)`